config/default/kustomization.yaml

```diff
@@ -16,6 +16,7 @@ bases:
 - ../crd
 - ../rbac
 - ../manager
+- ../snapshotgc
```

Comment:
can we not include it as default kustomization? I'd like to test it on stage longer and not promote it into production quickly (respectively, not blocking prod promotions of other features).

Once it's well tested, we can make it part of default kustomization

Reply:
Do you want to have there also the rbac-related stuff, or we keep those under rbac as part of the defaults?

Reply:
rbac is fine as is, it won't break anything if created :D